cmd/keymaster/agent.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"runtime"
+
+	"golang.org/x/crypto/ssh"
+	"golang.org/x/crypto/ssh/agent"
+
+	"github.com/Cloud-Foundations/Dominator/lib/log"
+	"github.com/cviecco/npipe"
```

Comment:
Please don't add a dependency on a personal repository.

Reply:
I can fork it into cloud-foundations. Would this work for you?

Reply:
That would be better. Is the upstream maintainer non-responsive?

Reply:
Seems so, there is no activity for the las 3 years and pull requests waiting. For now a fork withing cloud-foundations seems better. The only change that I did was make a stub for all non-windows OS. Want to review that?  Of should I just land?

Reply:
Also, while we agree on this there will be another push with this item pending.